backend/src/main/java/COMP_49X_our_search/backend/database/repositories/ProjectRepository.java

```diff
@@ -0,0 +1,10 @@
+package COMP_49X_our_search.backend.database.repositories;
+
+import COMP_49X_our_search.backend.database.entities.Project;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ProjectRepository extends JpaRepository<Project, Integer> {
+  List<Project> findAllByDepartments_Id(Integer departmentId);
+  List<Project> findAllByMajors_Id(Integer majorId);
```

Comment:
I think we may eventually want to make a method to findAllByUmbrellaTopics_Id and findAllByResearchPeriod_Id as well for the filtering requirements. But I'm thinking you were planning to implement those functions in a later iteration (because our first user story doesn't involve filtering). Lmk

Reply:
Yes, we'll need those in the future.